### ISSUE TITLE
disable registrations documentation doesn't follow default c2s.xml

### ISSUE THREAD

**Issue by brianjmurrell** [jabberd2/jabberd2#122]
Section 4.6.2 of [the documentation](https://github.com/jabberd2/jabberd2/wiki/InstallGuide-BasicConfiguration) on how to disable registrations refers users to the _Registration configuration subsection under the Authentication/registration database configuration_, yet the default [c2s.xml](https://github.com/jabberd2/jabberd2/blob/master/etc/c2s.xml.dist.in) doesn't seem to have any such section.  In fact none of the documentation in that section relates to any of the default content of _c2s.xml_ at all.  The **<register>** tag doesn't exist anywhere in that file.

So two things...

1. It would be nice if the default file contents more reflected the documentation (or vice versa)
2. Where am I supposed to put the <register></register> tags in the default c2s.xml (as referred to above) file?

**Comment by smokku** [comment]
Yeah... Unfortunately the _Install Guide_ is ancient. It was created for jabberd 2.0 and hardly updated.
The real, current documentation are te comments in the configuration files.

As C2S got support for virtual hosting, these configuration options were moved to **local.id** attributes: https://github.com/jabberd2/jabberd2/blob/b8e2095/etc/c2s.xml.dist.in#L127

**Comment by brianjmurrell** [comment]
OK.  Good.  This is making more sense.  However, I have _register-enable_ set to _false_ and I am still getting spammer registrations.  I.e.

`<id realm='example.com' register-enable='false' pemfile='/etc/jabberd/server.pem'>example.com</id>`

**Comment by smokku** [comment]
Please read the comment I linked

> Remove this attribute to disable account registrations.

You need to remove the attribute. The actual value does not matter.

**Comment by brianjmurrell** [comment]
Ahh.  OK.  I will give that a try.  Thanks.
